diff --git a/deps/stone/stone_mod_install.sh b/deps/stone/stone_mod_install.sh
index b26ee3a..12c6d45 100644
--- a/deps/stone/stone_mod_install.sh
+++ b/deps/stone/stone_mod_install.sh
@@ -14,7 +14,7 @@ vg0=$(printf t2-$VERSION_ID-%x $RANDOM)
 
 # detect platform once
 platform=$(uname -m)
-platform2=$(grep '\(platform\|type\)' /proc/cpuinfo)
+platform2=$(grep '\(platform\|type\)' /proc/cpuinfo || true)
 platform2=${platform2##*: }
 platform2=${platform2:-pc}
 [ -e /sys/firmware/efi ] && platform="$platform-efi" ||
